sidekick_core/lib/src/update/patches/157_fix_usage_message.patch.dart

```diff
@@ -15,8 +15,8 @@ final _gitPatch1 = MigrationStep.gitPatch(
 
 String _patch1() {
   final cliMainFilePath = relative(
-    Repository.requiredSidekickPackage.cliMainFile.absolute.path,
-    from: findRepository().root.path,
+    SidekickContext.sidekickPackage.cliMainFile.absolute.path,
+    from: SidekickContext.projectRoot.path,
```

Comment:
This is a git operation. It requires from to be the actual git root.

Maybe we should keep a way to query it?

Reply:
Just tested: `git apply` works without a git repository!

Reply:
Fixed 27576d4d3683f483c90dd090d2d315505a5faffb